packages/oui-collapsible/src/collapsible.controller.js

```diff
@@ -0,0 +1,19 @@
+import { addBooleanParameter } from "@oui-angular/common/component-utils";
+
+export default class {
+    constructor ($attrs, $element, $timeout) {
+        "ngInject";
+
+        this.$attrs = $attrs;
+        this.$element = $element;
+        this.$timeout = $timeout;
```

Comment:
`this.$element` and `this.$timeout` are not used. We can remove them.
But we still need `this.$attrs` for addBooleanParameter`addBooleanParameter.

Reply:
Now the component accepts an id, which is required for aria-controls attribute. This id must be removed from the original directive and hence these dependencies are now required :)